### DISCUSSION TITLE
Double charge, how to get refund

### DISCUSSION THREAD

**Discussion by Snazz2001** [community/community#123285] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Github team,

 I'm in the situation where I've been charged twice for Github Copilot (maybe my operation error). Can I have the money refund? How long usually it takes for response? Thanks in advance! The ticket is [here](https://support.github.com/ticket/personal/0/2778557)

**Comment by LiteBrite82** [comment]
Hi @Snazz2001 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
